core/server/common/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -116,6 +142,7 @@
   private static Map<String, Collection<ValidationTask>> initializeTargetTasks() {
     Map<String, Collection<ValidationTask>> targetMap = new TreeMap<>();
     ValidationTask[] commonTasks = {
+        HDFS_VALIDATION_TASK,
```

Comment:
why not add to the respective list (common/master/worker) as part of registerTask?

Reply:
@bf8086 
What do you think?

Reply:
@madanadit I prefer it this way mostly because it is easier to see what are included for master or worker. Are there any disadvantages?

Reply:
if someone who is not familiar with the code adds a new task and registers it, but forgets to add it to one of these lists will it not run?. i felt having registerTask handle it might be more maintainable. but it's up to you guys.

Reply:
That's a good point. I am fine with the change.

Reply:
Done. Will push the changes shortly.